hack/tools.go

```diff
@@ -19,10 +19,18 @@ limitations under the License.
 
 package tools
 
-// This package imports things required by this repository, to force `go mod` to see them as dependencies
+// This package imports things required by this repository, to force
+// `go mod` to see them as dependencies.
+//
+// https://github.com/golang/go/wiki/Modules#how-can-i-track-tool-dependencies-for-a-module
 import (
+	_ "github.com/pivotal-cf/yaml-patch/cmd/yaml-patch"
+
 	_ "k8s.io/code-generator/cmd/client-gen"
 	_ "k8s.io/code-generator/cmd/deepcopy-gen"
 	_ "k8s.io/code-generator/cmd/informer-gen"
 	_ "k8s.io/code-generator/cmd/lister-gen"
+	_ "sigs.k8s.io/controller-tools/cmd/controller-gen"
```

Comment:
we cannot do this. We must not have a dependency on external tools that depend on kube libraries. It will block rebases.